Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:argon
+
+# Install node_modules into a different directory to avoid npm/npm#9863.
+RUN mkdir -p /srv/addons-frontend-node
+ADD package.json /srv/addons-frontend-node
+WORKDIR /srv/addons-frontend-node
+RUN npm install -g npm@3
+RUN npm install
+
+ADD . /code
+WORKDIR /code
+
+# Replace the local node_modules with the ones we installed above.
+RUN rm -rf node_modules
```

Comment:
I see - this won't typically be needed in a docker hub build but you need this when building from your local src.